src/content/configKinds/cargo-config-toml.yaml

```diff
@@ -0,0 +1,30 @@
+---
+name: .cargo/config.toml
```

Comment:
Oh i missed how there are two very similar kinds. But I notice they both have the same filepath in their kinds? Should these maybe be one type? I don't know much about rust/cargo

Reply:
The issue I found here is that project configuration is in a `Cargo.toml` file in the root of the project.

The actual configuration for the Cargo utility is in a `.cargo/config.toml` file and the two are exclusive, i.e. you can't mix configuration options in a `Cargo.toml` that belong in a `.cargo/config.toml`. So, I built two separate collections since it seemed like we were keying off the file name or configuration type.

Happy to rethink the way we're presenting Rust, though!